cluster-scope/overlays/ocp-staging/groups/cluster-admins.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: user.openshift.io/v1
```

Comment:
You're not referencing this file anywhere. It should be a patch.

Reply:
right, it's there now.